src/pages/Popup/index.tsx

```diff
@@ -18,7 +17,12 @@ import '../../shared/font.css';
 
 import { logPageView, logSearchClick } from '../../shared/logs';
 
-function Panel() {
+interface PopupPanelProps {
+    hideAlpha?: boolean;
+    hideSettings?: boolean;
+}
+
+export function PopupPanel({ hideAlpha = false, hideSettings = false }: PopupPanelProps) {
```

Comment:
I'm missing something, is 'hideAlpha' always false?

Reply:
It's a default value, so `<PopupPanel />` means it is false but `<PopupPanel hideAlpa />` is true